src/slices/filters.ts

```diff
@@ -27,9 +27,6 @@ const reducers = {
   setFacets: (state: FiltersState, action: PayloadAction<DisplayableFacet[]>) => {
     state.facets = action.payload;
   },
-  setSortBys: (state: FiltersState, action: PayloadAction<SortBy[]>) => {
```

Comment:
I think we should keep sorts in here and remove them from the `vertical` state. The `filters` slice of state was meant to represent facets, filters, and sorts. @cea2aj or @oshi97, do you know why sorts appear in the `vertical` state chunk too?

Reply:
I'm not aware of any reason for them to be in both slices, and I agree that it should only be in one of them. I spoke with Nidhi about it, and she said that having the sorts in the filters slice was potentially confusing because it almost looks like the filters themselves are being sorted. We were thinking of keeping them in the `vertical` state to avoid that confusion